### PR TITLE
Made a few minor changes to postgres_writer.

### PR DESCRIPTION
**Pull request by anayanapalli**
@emilymendelson Can you please test these changes when you are testing the code next time? Also please go through the minor changes that I made.

### CODE REVIEW

File: utils/postgres_writer.py [hunk 1]
```diff
@@ -11,17 +12,16 @@ def insert_postgres_table_if_rows_not_exist(
...
-    - unique_key_columns (list): Columns that will prevent data insert on conflict.
+    Args:
+        dataframe (pd.DataFrame): The DataFrame to be written.
+        table_name (str): The name of the PostgreSQL table.
+        engine (sqlalchemy.engine.base.Engine): SQLAlchemy engine for database connection.
+        unique_key_columns (list): Columns that will prevent data insert on conflict.
 
     Returns:
-    - None (or Error)
+        None (or Error)
```
Comment (anayanapalli) [inline utils/postgres_writer.py:22]: Can you be a bit more specific as it is a None when successful and the Error string if not?

File: utils/postgres_writer.py [hunk 2]
```diff
@@ -38,11 +38,11 @@ def insert_postgres_table_if_rows_not_exist(
         )
 
         # Create a SQL INSERT statement with ON CONFLICT DO NOTHING clause
-        insert_sql = f"INSERT INTO {table_name} ({column_names}) VALUES {data_to_insert} ON CONFLICT ({', '.join(unique_key_columns)}) DO NOTHING;"
+        insert_sql = f"INSERT INTO {table_name} ({column_names}) VALUES %s ON CONFLICT ({', '.join(unique_key_columns)}) DO NOTHING;"
 
         # Execute the SQL statement with parameter binding
         with engine.begin() as conn:
-            conn.execute(text(insert_sql))
+            conn.execute(text(insert_sql), data_to_insert)
```
Comment (anayanapalli) [inline utils/postgres_writer.py:45]: I want to see how this behaves. May be it will break...

File: utils/postgres_writer.py [hunk 3]
```diff
@@ -11,17 +12,16 @@ def insert_postgres_table_if_rows_not_exist(
...
 
-    Parameters:
-    - dataframe (pd.DataFrame): The DataFrame to be written.
-    - table_name (str): The name of the PostgreSQL table.
-    - engine (sqlalchemy.engine.base.Engine): SQLAlchemy engine for database connection.
-    - unique_key_columns (list): Columns that will prevent data insert on conflict.
+    Args:
+        dataframe (pd.DataFrame): The DataFrame to be written.
+        table_name (str): The name of the PostgreSQL table.
+        engine (sqlalchemy.engine.base.Engine): SQLAlchemy engine for database connection.
```
Comment (anayanapalli) [inline utils/postgres_writer.py:18]: Sorry we need to make everything `Args` in all the files to be consistent.

File: utils/postgres_writer.py (hunk 2 above)
Comment (emilymendelson) [reply utils/postgres_writer.py:45]: @anayanapalli Tested this locally, but it does break the code ('Error: (psycopg2.errors.SyntaxError) syntax error at or near "%"')

**Pull request closed without merging**